### ISSUE TITLE
Music not playing

### ISSUE THREAD

**Issue by kishanverma26** [wlanjie/trinity#31]
Some time when i select music after that i go for recording while video recording music is not playing in background  and it crash some times

**Comment by wlanjie** [comment]
Please commit log file

**Comment by kishanverma26** [comment]
[trinity.zip](https://github.com/wlanjie/trinity/files/4785685/trinity.zip)
trinity.mmap3  this file logs has been created

**Comment by wlanjie** [comment]
No, xlog files under /sdcard/trinity are required, not mmap3 files

**Comment by kishanverma26** [comment]
[trinity_20200619.zip](https://github.com/wlanjie/trinity/files/4802350/trinity_20200619.zip)
here it is 
it crash some time while recoding videos

**Comment by wlanjie** [comment]
If it is a crash, I need the phone model, date and time information of the crash, and the log file does not record the crash information

**Comment by iamkdblue** [comment]
@kishanverma26 did you find any solution ?
i am getting same problem, music is not playing.

**Comment by monusk111** [comment]
> If it is a crash, I need the phone model, date and time information of the crash, and the log file does not record the crash information

I am using MI redmi note 4 and honor 6x for testing

**Comment by ThinkstartAndroidProject** [comment]
Some time when i select music after that i go for recording while video recording music is not playing in background and it crash some times

**Comment by amitkumar96** [comment]
Some time when i select music after that i go for recording while video recording music is not playing in background and it crash some times and Version 7.0 Device name MI Note 4
